### PR TITLE
Implement additional changes in station and delivery properties dialog.

### PR DESCRIPTION
**Pull request by birgitlewicki**
Ticket: #421

Changes as described in the comment:
https://github.com/SiLeBAT/fcl-client/issues/421#issuecomment-1239042234

### CODE REVIEW

File: src/app/tracing/dialog/station-properties/station-properties.component.ts [hunk 1]
```diff
@@ -149,6 +149,10 @@ export class StationPropertiesComponent implements OnInit, OnDestroy {
     notListedProps: string[] = ['name', 'incoming', 'outgoing'];
     otherProperties: string[] = [];
```
Comment (ruegenbfr) [inline src/app/tracing/dialog/station-properties/station-properties.component.ts:151]: Please replace name `specialProps` by `conditionalNotListedPropsMap`.
and 
`station.isMeta === true` by 'station.isMeta === false`

File: src/app/tracing/dialog/station-properties/station-properties.component.ts [hunk 2]
```diff
@@ -211,7 +215,7 @@ export class StationPropertiesComponent implements OnInit, OnDestroy {
     private initProperties(station: StationData, columns: TableColumn[]): void {
         const properties: Properties = {};
         const hiddenProps = Utils.createSimpleStringSet(this.notListedProps);
```
Comment (ruegenbfr) [inline src/app/tracing/dialog/station-properties/station-properties.component.ts:217]: Please add row: 
`this.conditionalNotListedPropsMap.forEach((isHiddenFun, propId) => hiddenProps[propId] = isHiddenFun(station));`
Please remove:
`&& !this.specialProps.has(key)`

File: src/app/tracing/dialog/station-properties/station-properties.component.ts [hunk 3]
```diff
@@ -222,6 +226,18 @@ export class StationPropertiesComponent implements OnInit, OnDestroy {
                 };
             });
```
Comment (ruegenbfr) [inline src/app/tracing/dialog/station-properties/station-properties.component.ts:228]: Remove added block.

**Review by ruegenbfr — CHANGES_REQUESTED** [review]

File: src/app/tracing/dialog/station-properties/station-properties.component.ts (hunk 1 above)
Comment (birgitlewicki) [reply src/app/tracing/dialog/station-properties/station-properties.component.ts:151]: done!

File: src/app/tracing/dialog/station-properties/station-properties.component.ts (hunk 2 above)
Comment (birgitlewicki) [reply src/app/tracing/dialog/station-properties/station-properties.component.ts:217]: done!

File: src/app/tracing/dialog/station-properties/station-properties.component.ts (hunk 3 above)
Comment (birgitlewicki) [reply src/app/tracing/dialog/station-properties/station-properties.component.ts]: done!

**Review by ruegenbfr — APPROVED** [review]
Looks good!

**Pull request merged**